### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change

This change adds some standard 10up open source details to the repo.

### Alternate Designs

none.

### Benefits

Provides a similar experience for users coming to any 10up open source repo.

### Possible Drawbacks

none identified.

### Verification Process

Manually verified via GitHub web UI.

### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

Relates to #141.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -187,6 +206,10 @@ WP Local Docker is an ever-evolving tool, and it's important to keep your local
 
 It's especially important to `git pull` the latest code before you `docker pull` upgrades to your Docker images, either as a potential fix for an issue or just to make sure they’re running the latest versions of everything. This will make sure you have the latest WP Local Docker code first, including the `docker-compose.yml` file that defines what Docker images and versions the environment uses.
 
+## Support Level
+
+**Active:** 10up is actively working on this, and we expect to continue work for the foreseeable future including keeping tested up to the most recent version of WordPress.  Bug reports, feature requests, questions, and pull requests are welcome.
```
Comment (cmmarslender) [inline readme.md]: @jeffpaul Lets put stable for this repo (v2 should be active)

File: readme.md [hunk 2]
```diff
@@ -2,9 +2,27 @@
 
 # WordPress Docker Development Environment
 
-This is a Docker based local development environment for WordPress.
-
-## What's Inside
+> This is a Docker based local development environment for WordPress.
+
+[![Support Level](https://img.shields.io/badge/support-active-green.svg)](#support-level)
```
Comment (cmmarslender) [inline readme.md]: @jeffpaul License should be MIT on this

**Review by cmmarslender — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (jeffpaul) [reply readme.md]: done, see: https://github.com/10up/wp-local-docker/pull/142/commits/3b7a59f635021fa1a00886477d2edd93ddbd35c3

File: readme.md (hunk 2 above)
Comment (jeffpaul) [reply readme.md]: done, see: https://github.com/10up/wp-local-docker/pull/142/commits/090d6e5e55ac0e8a025b9b63c3855864fbe02a40 && https://github.com/10up/wp-local-docker/pull/142/commits/df1abf1e4980eedc057addcca4f18996986cec0e

**Comment by jeffpaul** [comment]
@cmmarslender back to you for re-review

**Pull request merged**